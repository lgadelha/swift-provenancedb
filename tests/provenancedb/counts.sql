select count(*) from dataset_containment;
select count(*) from dataset_filenames;
select count(*) from dataset_usage;
select count(*) from executes;
select count(*) from executes_in_workflows;
select count(*) from invocation_procedure_names;
select count(*) from known_workflows;
select count(*) from workflow_events;

