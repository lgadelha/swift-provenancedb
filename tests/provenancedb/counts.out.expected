505
629
419
226
0
259
145
144
